shared/constants/types/chat2/index.tsx

```diff
@@ -167,6 +173,7 @@ export type _State = {
   createConversationError: string | null
   threadLoadStatus: I.Map<Common.ConversationIDKey, RPCChatTypes.UIChatThreadStatus>
   dismissedInviteBannersMap: I.Map<Common.ConversationIDKey, boolean>
+  lastCoord: Coordinate | null
```

Comment:
Maybe just use a zero accuracy/lat/lon as empty instead of `null`? Can have this const and compare against it to test if it's populated.

```
const emptyCoordinate = makeCoordinate()
```